tensorflow/compiler/mlir/hlo/lib/Dialect/gml_st/IR/gml_st_ops.cc

```diff
@@ -1734,7 +1734,7 @@ LogicalResult TransposeDimsOp::verify() {
   size_t rank = permutation().size();
   SmallVector<int64_t> position(rank, -1);
   for (const auto &it : llvm::enumerate(permutation())) {
-    int64_t dim = it.value();
+    size_t dim = it.value();
```

Comment:
I rather keep signed integer semantics whenever possible, through casting as needed: all int64_t instead of size_t are really intended.

Reply:
Makes sense, thanks!  I updated the patch to prefer signed integers over size_t.